html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get">
+    <div>
+      <label for="email">Email</label>
+    </br>
+      <input type="email" id="email" name="email" maxlength="12">
+    </div>
+    <div>
+      <label for="why-is-coding-cool?">Why is coding cool?</label>
+    </br>
+      <select name="options">
```

Comment:
don't forget the `id` attribute here which matches the `for` attribute on the label